pages/partials/submitRateFooter.ejs

```diff
@@ -0,0 +1,26 @@
+<% if (showGradeButton && typeof assessment_question !== "undefined" && assessment_question.grade_rate_minutes) { %>
+<div class="row">
+  <div class="col d-flex justify-content-between">
+  <span class="d-flex">
+  <% if (disableGradeButton) { %>
+    <small class="font-italic ml-2 mt-1 submission-suspended-msg">Grading possible in <span id='submission-suspended-display'></span> <div id='submission-suspended-progress' class='border border-info'></div> </small>
+    <%- include('countdown', {
+         serverTimeLimitMS: assessment_question.grade_rate_minutes * 60 * 1000,
+         serverRemainingMS: instance_question.allow_grade_left_ms,
+         progressSelector: '#submission-suspended-progress',
+         displaySelector: '#submission-suspended-display',
+         serverUpdateURL: false,
+         timerOutCode: "window.nextSubmissionReady();",
+         backgroundColorFunction: false,
+        }) %>
+  <% } %>
+  </span>
+  <span class="d-flex align-self-center">
+    <small class="font-italic">Can only be graded once every <%= assessment_question.grade_rate_minutes %> minute<% if (assessment_question.grade_rate_minutes > 1) { %>s<% } %></small>
+    <a class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<p>This question has been set up to limit the rate at which you receive feedback for your work to no more than once every <%= assessment_question.grade_rate_minutes %> minute<% if (assessment_question.grade_rate_minutes > 1) { %>s<% } %>. This means that, once you select <strong>Save &amp; Grade</strong>, you will be unable to use the same option again for this amount of time.</p><p>This limit does not affect your ability to Save your submission. You may save a submission at any time. Instructors have the ability to grade your saved submission, which will take the time you saved your answer into account to account for deadlines and credit. <% if (question_context == "student_exam") { %>A saved submission will also be graded automatically once you close the assessment. <% } %></p>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
```

Comment:
I'm trying to use "your answer" rather than "submission" when talking to students. I felt like the details about what instructors can do are not really helpful for students?
```suggestion
    <a class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<p>Once you have clicked <strong>Save &amp; Grade</strong>, you will need to wait <%= assessment_question.grade_rate_minutes %> minute<% if (assessment_question.grade_rate_minutes > 1) { %>s<% } %> before you can grade again.</p><p>You can still save your answer as frequently as you like. <% if (question_context == "student_exam") { %>If your assessment ends before your last saved answer is graded, it will be automatically graded for you.<% } %></p>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
```